src/ripple/test/jtx/impl/Env.cpp

```diff
@@ -119,7 +124,15 @@ Env::close(NetClock::time_point closeTime,
     // Round up to next distinguishable value
     closeTime += closed()->info().closeTimeResolution - 1s;
     bundle_.timeKeeper->set(closeTime);
-    app().getOPs().acceptLedger(consensusDelay);
+    // Go through the rpc interface unless we need to simulate
+    // a specific consensus delay.
+    if (consensusDelay)
+        app().getOPs().acceptLedger(consensusDelay);
```

Comment:
Agreed we should keep our eye on it. It just seemed a lot more reasonable to go this way than to try to pass a `std::chrono::duration` via RPC for the ~3 test cases that actually specify a time.